Berlin/PetermannIINachtrag42.xml

```diff
@@ -48,183 +147,419 @@ schematypens="http://relaxng.org/ns/structure/1.0"?>
+                     </decoNote>
+                  </decoDesc>
+                  
+                  <additions>
+                     <list>
+                        <item xml:id="e1">
+                           <desc>The manuscript contains many erasures and corrections above erasures, sometimes in condensed script, throughout.</desc>
+                        </item>
+                        
+                        <item xml:id="e2">
+                           <desc>Note in German on the front pastedown, presumably giving two dates on which the manuscript has been
+                              collated, and a signature.</desc>
+                           <q xml:lang="de">Kollationiert: fol. 1-298
+                           <date when="1933-02-25">25.2.1933</date>
+                           <date when="1935-10-05">5.10. 1935</date></q>
+                        </item>
+                        
+                        <item xml:id="e3">
+                           <locus target="#iv"/>
+                           <desc>Bookbinder's stamp in the lower outer edge of the paper guard leaf.</desc>
+                           <q xml:lang="de">W. Collin
+                              Kaiserl. u. Königl. Hofbuchbinder
+                              Berlin.</q>
+                        </item>
+                        
+                        <item xml:id="e4">
+                           <locus target="#iiv"/>
+                           <desc>Note in German.</desc>
+                           <q xml:lang="de"></q>
+                        </item>
+                        
+                        <item xml:id="e5">
+                           <locus target="#1r"/>
+                           <desc>Note by a European hand.</desc>
+                        </item>
+                        
+                        <item xml:id="e6">
+                           <locus target="#1r"/>
+                           <desc>Shelfmark.</desc>
+                           <q xml:lang="de">Pet. II Nachtr. 42.</q>
+                        </item>
+                        
+                        <item xml:id="e7">
+                           <locus target="#1r"/>
+                           <desc>Stamp of the library.</desc>
+                        </item>
+                        
+                        <item xml:id="a1">
+                           <locus target="#1r"/>
+                           <desc>Note listing the main content of the manuscript.</desc>
+                           <q xml:lang="gez">ዝ፡ መጽሐፍ፡ ኢሳይያስ፡ ወኤርምያስ፡ ወባሮክ፡ ወሕዝቅኤል።</q>
+                        </item>
+                        
+                        <item xml:id="a2">
+                           <locus target="#220v"/>
+                           <desc type="Doxology">Crude note written below the end of the main text.</desc>
+                           <q xml:lang="gez">ይትባርክ፡ እግዚአብሔር፡ አምላከ፡ እስራኤል፡ አምላከ ለኩሉ፡ መንፈስ፡ <sic>እግዚ፡</sic> ወለኵሉ፡ ዘስጋ፡</q>
```

Comment:
Should there be a "sic" or similar for the ር in ይትባርክ፡? Also, the ራ in እስራኤል፡ is discutible, but maybe there is no point in being too pedantic, when the intended meaning is clear…